src/controllers/api.controller.ts

```diff
@@ -215,3 +215,49 @@ export const nodesGet = async (
       passToExpressErrorHandler(err, next);
     }
   };
+
+    export const getNodeForLoggedInUser = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+  ) => {
+    try {
+      const userId = (req as ModifiedRequest).userId;
+      const rawClaim: any = turnFalsyPropsToUndefined(req.body);
+      
+      // Find a single node connected to the user's claims
+      const node = await prisma.node.findMany({
+        where: {
+          edgesTo: {
+            some: {
+              claim: {
+                issuerId: `http://trustclaims.whatscookin.us/users/${userId}`,
+                // `http://trustclaims.whatscookin.us/users/${userId}`
```

Comment:
this is fine, later we also want to find the nodes with their metamask DID - most of them will NOT be by their issuer id

Most would be by their DID, most users will NOT identify by our user id, but by some external universal way

We can use this for now but please add a comment to remidn people of this or add something ith their DID